src/MagicBall/MagicBall.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react';
+import BallWrapper from './BallWrapper';
+import InputWrapper from './InputWrapper';
+import HistoryWrapper from './HistoryWrapper';
+import './MagicBall.css'
+
+class MagicBall extends Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            title: 'Магический шар предсказаний',
+            answers: [
+                { color: 'blue', answer: 'Бесспорно' },
+                { color: 'blue', answer: 'Предрешено' },
+                { color: 'blue', answer: 'Никаких сомнений' },
+                { color: 'blue', answer: 'Определённо да' },
+                { color: 'blue', answer: 'Можешь быть уверен в этом' },
+                { color: 'green', answer: 'Мне кажется - "да"' },
+                { color: 'green', answer: 'Вероятнее всего' },
+                { color: 'green', answer: 'Хорошие перспективы' },
+                { color: 'green', answer: 'Знаки говорят  - "да"' },
+                { color: 'green', answer: 'Да' },
+                { color: 'yellow', answer: 'Пока не ясно, попробуй ещё' },
+                { color: 'yellow', answer: 'Спроси позже' },
+                { color: 'yellow', answer: 'Лучше не рассказывать' },
+                { color: 'yellow', answer: 'Сейчас нельзя предсказать' },
+                { color: 'yellow', answer: 'Сконцентрируйся и спроси опять' },
+                { color: 'red', answer: 'Даже не думай' },
+                { color: 'red', answer: 'Мой ответ - "нет"' },
+                { color: 'red', answer: 'По моим данным - "нет"' },
+                { color: 'red', answer: 'Перспективы не очень хорошие' },
+                { color: 'red', answer: 'Весьма сомнительно' },
+            ],
+            ballWrapper: 'Узнай свою судьбу',
+            inputValue: '',
+            historyWrapper: [],
+        }
+    }
+    handlerOnclick = () => {
+        if (this.state.inputValue) {
+            const random = Math.floor(Math.random() * this.state.answers.length)
```

Comment:
as for me better to move answers to constant because this state is never change

i mean outside from component

export const MAGIC_BALL_ANSWERS = [...]